dmapiclient/search.py

```diff
@@ -86,8 +86,8 @@ def get_frontend_params_from_search_api_url(self, search_api_url):
     def get_index_from_search_api_url(self, search_api_url):
         return self._url_reverse(search_api_url)[0]
 
-    def get_search_url(self, index, q=None, page=None, **filters):
-        return self.get_url(path='search', index=index, q=q, page=page, **filters)
+    def get_search_url(self, index, doc_type='services', q=None, page=None, **filters):
+        return self.get_url(path='search', index=index, doc_type=doc_type, q=q, page=page, **filters)
```

Comment:
You should either append the new `doc_type` arg onto the "end" of the args or add it as a breaking change to `CHANGELOG` as it could in theory be being used positionally by clients. (yeah, I know...)